webapp/app.py

```diff
@@ -38,27 +39,26 @@ def login_required(func):
 
     @functools.wraps(func)
     def is_user_logged_in(*args, **kwargs):
```

Comment:
I don't think this function is used anywhere - delete it?

Reply:
Do you mean the decorator or the is_user_logged_in?

Reply:
I mean the whole `is_user_logged_in` function:

``` bash
robin@rt480:~/Projects/anbox-cloud.io (invitation-code) $ rg is_user_logged_in
webapp/app.py
63:    def is_user_logged_in(*args, **kwargs):
70:    return is_user_logged_in
```

^ Only the function definition is listed

Reply:
BTW, the `rg` command I'm using is [RipGrep](https://snapcraft.io/ripgrep) - and it's awesome. An insanely fast file searching utility written in Rust I believe.

Reply:
I don't think I can delete it. It will break the `login_required` that goes through `/demo`